### DISCUSSION TITLE
Nodes Stop generating blocks

### DISCUSSION THREAD

**Discussion by harrybite** [cosmos/community#77] [Chain Builders (Cosmos SDK)]
I am using cosmos-sdk 0.50. 
I have changed the address prefix from cosmos to nikk (only wallet address)

I was running a node on the server and today I added a validator to my network, everything was working perfectly, and the second node was synced fully, but the main problem arose when I sent the create-validator tx with
 `./build/simd tx staking create-validator /Users/nikhil/Downloads/Github/Sovereign-T-Protocol-sdk-harry/validator.json --chain-id test-chain-2  --from nikkvalidator`
 
 node stops generating blocks and shows this error 
<img width="881" alt="Screenshot 2024-04-07 at 1 55 15 PM" src="https://github.com/cosmos/cosmos-sdk/assets/60419615/aa5326bb-9891-4de5-b01f-d5cc83d7e460">

**Comment by tac0turtle** [comment]
is the second up and running? if you spilt the power then the network is waiting for the other validator to come up
